VALUES-CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,23 @@
+Our code of conduct are the measures which ensure that we stick to our values
+at every stage of our work and all over at our organisation.
+
+
+## Values
+- ScaLabs is both labs & people
+- Functioning societies produce wonder
+- Human rights are universal
+- Good is green
+- Less but better
+- Form follows the content
+- Committment to highest standards defines us
+
+Code of conduct:
```

Comment:
Don't judge colleagues when they ask "stupid" questions. We want an open discourse-positive culture